lambda_functions/execute_ssh_command_js/serverless.yml

```diff
@@ -28,6 +28,7 @@ custom:
 provider:
   name: aws
   runtime: nodejs6.10
+  timeout: 300 # 5 minutes (max). Default is 6 seconds
```

Comment:
Would something between 6 and 300 be better? We don't want things to hang forever if they've gone into some sort of failure state. It's good to keep lambda executions as short as possible.

If we've got things that are anywhere near approaching 5 minutes, we probably should be running them in the background or in a PBS jobs, an then checking asynchronously for success/failure.